samples/sensor/bq34110/README.rst

```diff
@@ -0,0 +1,44 @@
+.. _bq34110-sample:
+
+BQ34110 Sensor Sample
+#####################
+
+Overview
+********
+
+This sample application retrieves all the fuel gauge parameters,
+- Voltage, in volts
+- Average current, in amps
+- Gauge temperature
+- State of charge measurement in percentage
+- Full Charge Capacity in mAh
+- Remaining Charge Capacity in mAh
+- Average power in W
+- Time to full in minutes
+- Time to empty in minutes
+
+from BQ34110 sensor and prints this information to the UART console.
+
+Requirements
+************
+
+This sample requires a `BQ34110 Fuel Gauge`_ and :dtcompatible:`ti,bq34110`
+overlay must be provided to connect the sensor to the I2C bus.
```

Comment:
```suggestion
This sample requires a board with a `BQ34110 Fuel Gauge`_ connected
via the I2C bus. The fuel gauge must be described with a devicetree node
with compatible :dtcompatible:`ti,bq34110`. If your board does not have 
this fuel gauge built-in, a devicetree overlay must be provided adding the node.
See :zephyr_file:`samples/sensor/bq34110/boards/frdm_k64f.overlay` for
an example devicetree overlay.
```

Reply:
README.rst file updated as per the suggestion.